components/role-mgt/org.wso2.carbon.identity.role.v2.mgt.core/src/main/java/org/wso2/carbon/identity/role/v2/mgt/core/RoleManagementService.java

```diff
@@ -441,13 +441,26 @@ Map<String, String> getMainRoleToSharedRoleMappingsBySubOrg(List<String> roleIds
             throws IdentityRoleManagementException;
 
     /**
-     * Get associated applications by role id.
+     * Get shared role to main role mappings by subOrg.
      *
-     * @param roleId       Role Id.
-     * @param tenantDomain Tenant Domain.
-     * @return List of application ids.
+     * @param roleIds            Shared role IDs.
+     * @param subOrgTenantDomain Sub Organization tenant domain.
+     * @return The map of shared role id to main roles.
      * @throws IdentityRoleManagementException IdentityRoleManagementException.
      */
+    default Map<String, String> getSharedRoleToMainRoleMappingsBySubOrg(List<String> roleIds, String subOrgTenantDomain)
+            throws IdentityRoleManagementException {
+        throw new NotImplementedException("getRoles method is not implemented");
+    }
+
+        /**
+         * Get associated applications by role id.
+         *
+         * @param roleId       Role Id.
+         * @param tenantDomain Tenant Domain.
+         * @return List of application ids.
+         * @throws IdentityRoleManagementException IdentityRoleManagementException.
+         */
```

Comment:
fix indentation of the comment

Reply:
Addressed and resolved in commit: [b2054ba](https://github.com/wso2/carbon-identity-framework/pull/6162/commits/b2054baf27f17ededa73e9b12c15530489faa707)